clang/include/clang/3C/ConstraintVariables.h

```diff
@@ -91,7 +104,7 @@ class ConstraintVariable {
   // Only subclasses should call this
   ConstraintVariable(ConstraintVariableKind K, std::string T, std::string N,
                      std::string D)
-    : Kind(K), OriginalType(T), Name(N), OriginalDecl(D),
+    : Kind(K), OriginalType(T), Name(N), OriginalTypeWithName(D),
```

Comment:
```suggestion
    : Kind(K), OriginalType(T), Name(N), OriginalTypeWithName(TN),
```
(And on the previous line, but GitHub doesn't seem to let me attach a suggestion to that line. :slightly_frowning_face:)